javascript-practice/src/js/views/userView.js

```diff
@@ -1,19 +1,58 @@
+import { validateEmail, validatePassword } from '../helpers/validation'
+import { ERROR_MESSAGE } from '../constants/message'
+
+export const {
+  REQUIRED_FIELD_EMAIL,
+  REQUIRED_FIELD_PASSWORD,
+  INVALID_EMAIL,
+  INVALID_PASSWORD,
+} = ERROR_MESSAGE
+
 export default class UserView {
   constructor() {
-    this.loginFormEl = document.getElementById('login-form');
+    this.loginFormEl = document.getElementById('login-form')
   }
 
   bindFormLogin = (submitLogin) => {
-    this.loginFormEl.addEventListener("submit", e => {
-      e.preventDefault();
-      const valueEmail = document.getElementById('email').value;
-      const valuePassword = document.getElementById('password').value;
-      
-      submitLogin(valueEmail, valuePassword); 
-    });
+    this.loginFormEl.addEventListener('submit', (e) => {
+      e.preventDefault()
+      const valueEmail = document.getElementById('email').value
+      const valuePassword = document.getElementById('password').value
+      const emailErrorEl = document.getElementById('email-error')
+      const passwordErrorEl = document.getElementById('password-error')
+
+      emailErrorEl.textContent = ''
+      passwordErrorEl.textContent = ''
+
+      if (!valueEmail || !valuePassword) {
```

Comment:
Condition is complicated and duplicated, please consider to refactor

Reply:
i done